src/main/java/seedu/address/model/room/Room.java

```diff
@@ -123,18 +124,29 @@ public Room checkIn() {
     }
 
     /**
-     * Checks out the first booking of this room and its current occupant.
```

Comment:
So guests are not deleted from Concierge after checking out?

Reply:
Not yet.